internal/controllers/user_devices_controller.go

```diff
@@ -1919,6 +1976,18 @@ type AdminRegisterUserDevice struct {
 	Verified    bool    `json:"verified"`
 }
 
+type GetUserDevicesByIntegration struct {
+	AuthorizationCode string `json:"authorizationCode"`
+	RedirectURI       string `json:"redirectUri"`
+	Region            string `json:"region"`
+}
+
+type GetUserDevicesByIntegrationResponse struct {
+	ID        int    `json:"id"`
+	VehicleID int    `json:"vehicle_id"`
```

Comment:
I think we decided to stick with camelCase
```suggestion
	VehicleID int    `json:"vehicleID"`
```

Reply:
`vehicleId`, I would think.